src/bots/slack/conversations/message.conversation.ts

```diff
@@ -0,0 +1,11 @@
+import { SlackBotWorker } from 'botbuilder-adapter-slack';
+import { BotkitMessage, Botkit } from 'botkit';
+
+export const onMessageConversation = async (controller: Botkit) => {
+  controller.on(
+    'message',
+    async (bot: SlackBotWorker, message: BotkitMessage) => {
+      await bot.reply(message, 'I heard a message!');
```

Comment:
Do we want to add a helpful message here? In the coffee bot, it responds to all random words with

```
I don't know what you mean. Type `order` to get started!"
```

Reply:
I'll make another PR to revise the convo's.